src/test/java/com/programmers/springweekly/domain/customer/CustomerTest.java

```diff
@@ -0,0 +1,22 @@
+package com.programmers.springweekly.domain.customer;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.UUID;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+
+public class CustomerTest {
+
+    @Test
+    @DisplayName("고객을 생성한다.")
```

Comment:
```suggestion
    @DisplayName("고객을 생성할수 있다.")
```

기대값에 대한 테스트가 이루어지면 좋겠어요. 그에 대한 assertion을 목표로 하구요.

Reply:
기대값에 대한 테스트가 이루어진다는게 어떤 의미인지 아직 정확하게 모르겠습니다! 조금만 더 자세하게 말씀해주실 수 있을까요? 🙏 

우선 DisplayName의 경우 모두 한다. 보다는 할 수 있다. 라고 네이밍 변경처리 했습니다! w2-4 PR에 종합적으로 반영시켰습니다!